public/app/core/components/NavBar/DropdownChild.tsx

```diff
@@ -17,10 +17,15 @@ const DropdownChild = ({ isDivider = false, icon, onClick, target, text, url }:
   const styles = getStyles(theme);
 
   const linkContent = (
-    <>
-      {icon && <Icon data-testid="dropdown-child-icon" name={icon} className={styles.icon} />}
-      {text}
-    </>
+    <div className={styles.linkContent}>
+      <div>
+        {icon && <Icon data-testid="dropdown-child-icon" name={icon} className={styles.icon} />}
+        {text}
+      </div>
+      {target === '_blank' && (
+        <Icon data-testid="external-link-icon" name="external-link-alt" className={styles.externalLinkIcon} />
+      )}
```

Comment:
nit: maybe dim the icon a bit? Now it's quite in-your-face and feels like it clutters the menu items a bit.

Reply:
agreed, great suggestion